### PR TITLE
backup: do fill in the file size of each SST file in the response

### PR DESCRIPTION
**Pull request by kennytm**
##  What have you changed?

In the backup service, the BackupResponse contains the metadata of every SST file being archived. One of the metadata is the file size, but it was not filled in. This PR added this back.

###  What is the type of the changes?

- Bugfix (a change which fixes an issue)

###  How is the PR tested?

- Unit test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

###  Does this PR affect `tidb-ansible`?

No

###  Refer to a related PR or issue link (optional)

pingcap/br#163

### CODE REVIEW

**Comment by kennytm** [comment]
PTAL @overvenus @5kbpers

**Comment by overvenus** [comment]
PTAL @fredchenbj , thanks!

**Comment by kennytm** [comment]
@overvenus The `size` field doesn't exist in release-3.1's kvproto.

https://github.com/pingcap/kvproto/blob/release-3.1/proto/backup.proto#L48-L62

**Comment by overvenus** [comment]
Could you cherry-pick it into kvproto release-3.1?

File: components/backup/tests/integrations/test_backup.rs [hunk 1]
```diff
@@ -621,15 +621,18 @@ fn test_backup_raw_meta() {
...
         checksum ^= f.get_crc64xor();
         total_kvs += f.get_total_kvs();
         total_bytes += f.get_total_bytes();
+        total_size += f.get_size();
     }
     assert_eq!(total_kvs, key_count + 1);
     assert_eq!(total_kvs, admin_total_kvs);
     assert_eq!(total_bytes, admin_total_bytes);
     assert_eq!(checksum, admin_checksum);
+    assert!(total_size > 0);
```
Comment (zhangjinpeng1987) [inline components/backup/tests/integrations/test_backup.rs]: I think we should use an exact size for this test.

Comment (kennytm) [reply components/backup/tests/integrations/test_backup.rs]:
```suggestion
    assert_eq!(total_size, 1611); // update this number if needed
```

@zhangjinpeng1987 the exact size relies on the SST encoding which can be considered a magic number. Is this change acceptable?

Comment (zhangjinpeng1987) [reply components/backup/tests/integrations/test_backup.rs]: For the specified version of RocksDB, it should be constant.

**Pull request merged**